src/test/ui/where-clauses/higher-ranked-fn-type.quiet.stderr

```diff
@@ -1,8 +1,8 @@
-error[E0277]: the trait bound `for<'b> for<'b> fn(&'b ()): Foo` is not satisfied
+error[E0277]: the trait bound `for<'b> fn(&'b ()): Foo` is not satisfied
   --> $DIR/higher-ranked-fn-type.rs:20:5
    |
 LL |     called()
-   |     ^^^^^^ the trait `for<'b> Foo` is not implemented for `for<'b> fn(&'b ())`
+   |     ^^^^^^ the trait `for<'b> Foo` is not implemented for `fn(&'b ())`
```

Comment:
While this does fix https://github.com/rust-lang/rust/issues/102414, it's kind of unfortunate that this omits the binder here. Not sure why that is yet.